scripts/core/EXP-USM-IMPORT-RELEASE.rst

```diff
@@ -0,0 +1,58 @@
+
+<%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+.. _experimental-usm-import-release:
+
+==================
+USM Import/Release
+==================
+
+.. warning::
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions over
+        time.
+    *   Do not require conformance testing of their own additions.
+
+
+Data transfer between Host and Device is most efficient when source and
+destination are both allocated in USM memory.
+In situations where host data will participate in host/device transfers
+and the host data allocation is under user control, USM functions
+such as malloc_host could be used to allocate USM memory instead of
+system memory.
+However, this is not always possible if the source code where the allocation
+is made is not available, or source code changes are prohibited for portability
+reasons.
+In these situations a mechanism to temporarily promote system memory to USM
+for the duration of the host/device data transfers is useful for maximizing
+data transfer rate.
+
+
+Import Host Memory into USM
+===========================
+
+Import a range of host memory into USM.
+
+.. parsed-literal::
+
+    // Import into USM
+    USMImport(hContext, hostPtr, size);

```

Comment:
In rst files use `${x}`/`${X}` for the prefix:

```suggestion
    ${x}USMImport(hContext, hostPtr, size);
```

Reply:
If I prefix the function name with "${x}" then the generate step emits this error message:
Generating ../docs/source/core/EXP-USM-IMPORT-RELEASE.rst...
/iusers/rdeodhar/cmplr/ur_l0import/llvm/scripts/core/EXP-USM-IMPORT-RELEASE.rst(45) : error : symbol '$xUSMImport' not found
/iusers/rdeodhar/cmplr/ur_l0import/llvm/scripts/core/EXP-USM-IMPORT-RELEASE.rst(57) : error : symbol '$xUSMRelease' not found
Failed to generate specification.

That's why I had left it out. Perhaps x needs to be defined somewhere. Not clear to me how/where.

Reply:
Another question about the generated files:
Are some of them read-modify-write, or are they all simply generated, i.e. written?

Reply:
> That's why I had left it out. Perhaps x needs to be defined somewhere. Not clear to me how/where.

Okay, I'll have a look at this and see if the generator script is doing something different for this file compared to the others.

>  Are some of them read-modify-write, or are they all simply generated, i.e. written?

So anything in the `include` and `source` directories which the generator scripts touch are simply generated. `scripts/core/registry.yml` is different and it read/modify/write but only when a function is added/updated/removed.

Reply:
> > That's why I had left it out. Perhaps x needs to be defined somewhere. Not clear to me how/where.
> 
> Okay, I'll have a look at this and see if the generator script is doing something different for this file compared to the others.
> 
> > Are some of them read-modify-write, or are they all simply generated, i.e. written?
> 
> So anything in the `include` and `source` directories which the generator scripts touch are simply generated. `scripts/core/registry.yml` is different and it read/modify/write but only when a function is added/updated/removed.

The generator script validation was actually correct, these are missing the `Exp` suffix. I'll make suggestions on your new PR.